package.json

```diff
@@ -11,10 +11,16 @@
     "url": "git+https://github.com/ngParty/Angular1-scaffold.git"
   },
   "scripts": {
-    "typings": "typings install",
-    "postinstall": "npm run typings",
-    "start": "webpack-dev-server --port $npm_package_config_port --hot --inline --progress --profile --colors --watch --display-error-details --display-cached --content-base src/",
-    "build": "NODE_ENV=production webpack --config webpack-build.config.js"
+    "rimraf": "rimraf",
+    "typings": "typings",
+    "postinstall": "npm run typings -- install",
+    "start": "npm run server",
+    "build": "NODE_ENV=production webpack --config webpack-build.config.js",
+    "build:dev": "webpack --progress --profile --colors --display-error-details --display-cached",
+    "test": "karma start",
+    "watch:test": "npm run test -- --auto-watch --no-single-run",
```

Comment:
I guess that it's better to have watch as last word as mainly you are testing then you are thinking about how to run the tests